index.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Ezio's CV</title>
+</head>
+<body>
+<header>
+    <h1>Portofolio</h1>
+    <nav>
+        <ul>
+            <li><a href="#about">About</a></li>
+            <li><a href="#biographical-information">Biographical information</a></li>
+            <li><a href="#political-information">Political information</a></li>
+            <li><a href="#gallery">Gallery</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+</header>
+<hr>
+<main>
+    <section id="about">
+        <h2>About</h2>
+        <img src="images/ezio_avatar.jpg" 
+        alt="Avatar image" 
+        width="200" 
+        height="200" 
+        title="Ezio">
+
+        <img src="images/auditore_crest.png" 
+        alt="Auditore crest image" 
+        width="200" 
+        height="200" 
+        title="Auditore crest">
+        <h3>Ezio Auditore da Firenze</h3>
+        <blockquote cite="https://assassinscreed.fandom.com/wiki/Ezio_Auditore_da_Firenze">
+        Ezio Auditore da Firenze (1459 – 1524) was a Florentine nobleman 
+        during the Renaissance, and, unbeknownst to most historians and 
+        philosophers, a Master Assassin and the Mentor of the Italian 
+        Brotherhood of Assassins, a title which he held from 1503 to 1513. H
+        e is also an ancestor of William and Desmond Miles, as well as 
+        Clay Kaczmarek.
+        </blockquote>
+    </section>
+    <hr>
+    <section id="biographical-information">
+        <h2>Biographical information</h2>
+        <ul>
+            <li>Born</li>
+                <ul>
+                    <li>24 June 1459 Florence, Republic of Florence</li>
+                </ul>
+                <li>Died</li>
+                <ul>
+                    <li>30 November 1524 (aged 65) Florence, Republic of Florence</li>
+                </ul>    
+                <li>Species</li>
+                <ul>
```

Comment:
Element **ul** not allowed as a child of element **ul** in this context.
In this context, the **ul** tag should only have **li** child tags

Reply:
Please make adjustments on all places in this document where "ul" tag is a child of "ul" tag

Reply:
Ok, thank you. As soon as i make adjustments, i will update the code.